documentation/docs/pages/docs/start-building/running.mdx

```diff
@@ -37,6 +37,151 @@ If you change your contract ABIs or want to start fresh you can use the [delete]
 You can also use the [drop_each_run](/docs/start-building/yaml-config/storage#drop_each_run) option in the YAML configuration file to drop all the data for the indexer before starting.
 :::
 
+## Health Monitoring
```

Comment:
this needs moving to its own file or something as running is easy and quick and this takes up a bit to much room and added context